cmd/vicadmin/vicadm.go

```diff
@@ -544,27 +575,26 @@ func (s *server) serve() error {
 	s.handleFunc("/container-logs.tar.gz", s.tarContainerLogs)
```

Comment:
Oh actually upon further inspection, this is `handleFunc` which does some things and then calls `HandleFunc`, confusing I know. I modified the call to `HandleFunc` inside of `handleFunc` so anything that calls the package-local `handleFunc` instead of `http.HandleFunc` gets authenticated; see line 563.